content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -308,6 +308,7 @@ The following networking functionality is not supported on Windows nodes
 * Host networking mode is not available for Windows pods
 * Local NodePort access from the node itself fails (works for other nodes or external clients)
 * Accessing service VIPs from nodes will be available with a future release of Windows Server
+* A single service can only support up to 64 backend pods / unique destination IPs
```

Comment:
An optional capitalization tweak:
```suggestion
* A single Service can only support up to 64 backend pods / unique destination IPs
```

Reply:
Thank you for the review @sftim, I was trying to keep it consistent with the rest of the bullets but I am happy to correct them and maybe change the others?

Reply:
The other uses seem to be talking about network services rather than instances of the Service resource. Anyway, a minor detail.